### PR TITLE
Update host-kernel

### PR DESCRIPTION
**Pull request by suqinhuang**
use Xm.Yn for host kernel now

### CODE REVIEW

**Comment by spcui** [comment]
Acked-by: scui@redhat.com

File: shared/cfg/host-kernel/Host_RHEL/7.cfg [hunk 1]
```diff
@@ -21,3 +21,7 @@
         black_cmds = "block-stream block-job-cancel block-job-set-speed drive-mirror block-job-complete block-job-pause block-job-resume"
     monitor_cmds_check.human:
         black_cmds = "block_stream block_job_cancel block_job_set_speed drive_mirror block_job_complete block_job_pause block_job_resume"
+    multi_queues_test.with_netperf.max_queues:
+        queues = 256
+    multi_queues_test.invalid_queues_number.upper_border:
+        queues = 257
```
Comment (CongSmile) [inline shared/cfg/host-kernel/Host_RHEL/7.cfg]: Could you show me the id for the above lines?

Thanks.

Comment (suqinhuang) [inline shared/cfg/host-kernel/Host_RHEL/7.cfg]: Hi,
What do you mean id? 
all the docs are copied from host-kernel in tp-qemu

Comment (CongSmile) [inline shared/cfg/host-kernel/Host_RHEL/7.cfg]: Bug id, thanks.

**Comment by suqinhuang** [comment]
ID: 1302551

**Comment by xutian** [comment]
@PandaWei  @will-Do  and @Hao-Liu   does these configuration file need by your test.  If not, let me move it to tp-qemu repo.

**Comment by suqinhuang** [comment]
let's merge it first, and remove it when internal is ready.

**Pull request merged**